### PR TITLE
Refactor README

### PR DESCRIPTION
**Pull request by IvanIvanoff**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,47 +1,39 @@
-# sanpy
```
Comment (spiderjako) [inline README.md:1]: Why is the title removed tho?

Comment (IvanIvanoff) [reply README.md:1]: not sure

Comment (IvanIvanoff) [reply README.md:1]: brought it back

File: README.md [hunk 2]
```diff
@@ -79,36 +74,125 @@ In order to access real-time data or historical data for some of the metrics,
...
-To obtain an api key you should [log in to sanbase](https://app.santiment.net/login)
-and go to the `account` page - [https://app.santiment.net/account](https://app.santiment.net/account).
+To obtain an API key you should [log in to sanbase](https://app.santiment.net/login)
+and go to the `Account` page - [https://app.santiment.net/account](https://app.santiment.net/account).
 There is an `API Keys` section and a `Generate new api key` button.
 
-If the account used for generating the api key has enough SAN tokens, the api key will give you
-access to the data that requires SAN token staking. The api key can only be used to fetch data and not to execute graphql mutations.
+If the account used for generating the api key has enough SAN tokens, the api
+key will give you access to the data that requires SAN token staking. The api
```
Comment (tspenov) [inline README.md:91]: I think that part for staked SAN tokens is no longer valid

**Review by tspenov — APPROVED** [review]

**Review by spiderjako — APPROVED** [review]

**Pull request merged**